maps/src/main/java/ar/com/wolox/wolmo/maps/fragment/AbstractMapFragment.java

```diff
@@ -0,0 +1,278 @@
+package ar.com.wolox.wolmo.maps.fragment;
+
+import android.support.annotation.CallSuper;
+import android.support.annotation.ColorRes;
+import android.support.annotation.DrawableRes;
+import android.support.annotation.NonNull;
+import android.support.annotation.Nullable;
+import android.support.annotation.StringRes;
+
+import com.google.android.gms.maps.CameraUpdate;
+import com.google.android.gms.maps.GoogleMap;
+import com.google.android.gms.maps.MapView;
+import com.google.android.gms.maps.OnMapReadyCallback;
+import com.google.android.gms.maps.model.BitmapDescriptorFactory;
+import com.google.android.gms.maps.model.LatLng;
+import com.google.android.gms.maps.model.LatLngBounds;
+import com.google.android.gms.maps.model.Marker;
+import com.google.android.gms.maps.model.MarkerOptions;
+import com.google.android.gms.maps.model.Polyline;
+import com.google.android.gms.maps.model.PolylineOptions;
+
+import java.util.List;
+
+import ar.com.wolox.wolmo.core.presenter.BasePresenter;
+import ar.com.wolox.wolmo.maps.R;
+
+/**
+ * Fragment that allows rendering {@link Marker}s and {@link Polyline}s on the {@link GoogleMap}. It
+ * can display them as 'selected', giving them a highlighted fashion.
+ *
+ * The rendering methods <b>have</b> to be called <b>after</b> {@link #onMapReady(GoogleMap)}, which
+ * is when the {@link MapView} finishes being laid out.
+ */
+public abstract class AbstractMapFragment<P extends BasePresenter & IMapPresenter> extends WoloxMapFragment<P> {
```

Comment:
I'm not sure if this class name is very descriptive, what about naming it something like `WolmoDrawableMapFragment`.
Just for your information, we should start renaming everything that says `Wolox` to `Wolmo`, to be consistent with the framework :)